xcode.md

```diff
@@ -12,165 +12,166 @@ SHA1 checksums of Apple's installers for:
 
 Strong cryptography makes us all safer.
 
-## How do I verify SHA1 checksums?
+## How can I help?
```

Comment:
This upper portion with the verbiage changes is unreverted. Can we isolate it to a separate commit?

(I don’t want to merge your changes as-is with the giant table redo and revert, and I don’t want to squash into a single commit with this change unrelated to Xcode 13 present.)